test/unit/shiki/input.js

```diff
@@ -0,0 +1,3 @@
+const javascriptGrammar = require('shiki/languages/javascript.tmLanguage.json');
```

Comment:
Similarly here, it should be the normal usage.

Also you need to add an output file for unit tests to compare the traced files include the language.

Reply:
Oops didn't realize this should go in output. Updated!